### ISSUE TITLE
Older icons reappearing after 9.6.5

### ISSUE THREAD

**Issue by Donnnno** [Arcticons-Team/Arcticons#2145]
In my quest to round more corners, I may have overwritten some icons, like for example LibreTube:

![335870267-37b58bcd-75a2-4a80-b952-f670840a56a1](https://github.com/Arcticons-Team/Arcticons/assets/31142286/38df4c65-ff80-4281-8aba-b91964e0469c)

Please notify me if other icons are overwritten too

#2144

**Comment by kfidaw4ir** [comment]
![image](https://github.com/Arcticons-Team/Arcticons/assets/42912975/d97089b0-fb5c-4aff-aedd-1f686a6cc59b)
![image](https://github.com/Arcticons-Team/Arcticons/assets/42912975/90f729d7-4074-45f6-9cec-b69e54ae0dda)

Notion

Quick question, what led to the overwritten of the icons? Would love some hindsight on this. Thanks for your work as always!

**Comment by Donnnno** [comment]
I've started updating all square based icons a couple of weeks ago. Its border radius has been increased with 2px to 4px total.
But I had to go through all the icons and apply that action. The fastest way of doing that was going through the files that I used, which exists of 10 big files with all the icons combined. 
The problem: older app icons were on there too, but with so many changes it's hard to notice every change.
